packages/core/src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -19,6 +19,10 @@ export class SqliteQueryGeneratorTypeScript extends MySqlQueryGenerator {
     return `PRAGMA INDEX_LIST(${this.quoteTable(tableName)})`;
   }
 
+  getToggleForeignKeyChecksQuery(enable: boolean): string {
```

Comment:
Can we add unit tests for the generation of this query?

Reply:
Done